docs/releasing.md

```diff
@@ -59,76 +59,67 @@ CAPM3 uses [semantic versioning](https://semver.org).
-
-- For any subdirectory with `go.mod` in it (excluding `hack/tools` and
-  `hack/fake-apiserver`), create another Git tag with directory prefix, ie.
-  `git tag api/v1.x.y` and `git tag test/v1.x.y`. This enables the
-  tags to be used as a Go module version for any downstream users.
-
-  **NOTE**: Do not create annotated tags (`-a`, or implicitly via `-m` or `-s`)
-  for Go modules. Release notes expects only the main tag to be annotated,
-  otherwise it might create incorrect release notes. Push both of the tags to
-  `origin`.
-
-### Release notes
-
-Next step is to clean up the release note manually. Release note has been
-generated by the `release` action, do not click the `Generate release notes`
-button. In case there is issue with release action, you may rerun it via
-`Actions` tab, or you can `make release-notes` to get a markdown file with
-the release content to be inserted.
-
-- If release is not a beta or release candidate, check for duplicates, reverts,
-  and incorrect classifications of PRs, and whatever release creation tagged to
-  be manually checked.
+or if using existing repository, make sure origin is set to the fork and
+upstream is set to `metal3-io`. Verify if your remote is set properly or not
+by using following command `git remote -v`.
+
+- Fetch the remote (`metal3-io`): `git fetch upstream`
+This makes sure that all the tags are accessible.
+
+### Creating Release Notes
+
+- Switch to the main branch: `git checkout main`
+
+- Create a new branch for the release notes**:
+  `git checkout -b release-notes-1.x.x origin/main`
+
+- Generate the release notes: `RELEASE_TAG=v1.x.x make release-notes`
+   - Replace `v1.x.x` with the new release tag you're creating.
+   - This command generates the release notes here
+     `releasenotes/<RELEASE_TAG>.md` .
+
+- Next step is to clean up the release note manually.
+   - If release is not a beta or release candidate, check for duplicates,
+     reverts, and incorrect classifications of PRs, and whatever release
+     creation tagged to be manually checked.
    - For any superseded PRs (like same dependency uplifted multiple times, or
-     commit revertions) that provide no value to the release, move them to
+     commit revertion) that provide no value to the release, move them to
      Superseded section. This way the changes are acknowledged to be part of the
      release, but not overwhelming the important changes contained by the
      release.
-- If the release you're making is not a new major release, new minor release,
-  or a new patch release from the latest release branch, uncheck the box for
-  latest release.
-- If it is a release candidate (RC) or a beta release, tick pre-release box.
-- Save the release note as a draft, and have others review it.
+
+- Commit your changes, push the new branch and create a pull request:
+   - The commit and PR title should be 🚀 Release v1.x.y:
+      -`git commit -S -s -m ":rocket: Release v1.x.x"`
```

Comment:
```suggestion
      - `git commit -S -s -m ":rocket: Release v1.x.x"`
```
This is broken now, space should fix it.
![image](https://github.com/user-attachments/assets/c4d00cbf-8c9a-46e7-b5cd-fb6d5e0a15c9)